javascript/script.js

```diff
@@ -0,0 +1,185 @@
+// Assignment Code
+var generateBtn = document.querySelector("#generate");
+
+function generatePassword() {
+  // PLEASE ADD ALL YOUR CODE HERE
```

Comment:
Once you've completed your code, please remove any comments related to process. You can leave brief comments which describe what the code is doing.